Sorts/test/MergeSort.test.ts

```diff
@@ -1,12 +1,25 @@
 import { MergeSort } from "../MergeSort";
 
 describe("Merge Sort", () => {
-    it("generating array and comparing with sorted array", () => {
+    it("generating array with length 10 and comparing with sorted array", () => {
         let arrLen = 10;
-        let inBuiltSortArr = Array.from(Array(arrLen)).map(x=>Math.random());
+
+        let inBuiltSortArr = Array<number>(arrLen)
+        for (let i = 0; i < arrLen; i++) { inBuiltSortArr[i] = Math.random() }
+        let mergeSortArray = inBuiltSortArr;
```

Comment:
You must copy at some point, otherwise you're calling merge sort on the same sorted array

Reply:
I am copying from the inBuiltSortArr before applying the in-built sort functionality to it.
For ex - if inBuiltSortArr = [3, 1, 2]
                 mergeSortArr = [3, 1, 2] intially

Reply:
Where are you copying it? `let mergeSortArray = inBuiltSortArr;` does not copy the array and `.sort()` is in-place. You can use `mergeSortArray = inBuiltSortArr.slice()` to copy.

Reply:
Okay my bad, updated the file